### ISSUE TITLE
Improve escaping of slashes in `path`....

### ISSUE THREAD

**Issue by the-t-in-rtf** [zaggino/z-schema#130]
For deeply hierarchical structures, z-schema returns the name of the problematic variable under `params[0]`, and the path to the variable under `path`, as in:

```
{
  "code": "OBJECT_MISSING_REQUIRED_PROPERTY",
  "params": [
    "nested"
  ],
  "message": "Missing required property: nested",
  "path": "#/top"
}
```

For simple cases, you can convert the path variable to a hierarchy of segments using the slash character.  Z-schema supports escaping of the slash character in keys, presumably with this in mind.  However, only the first slash is escaped.  As an example, assume you have this schema:

```
{
    "$schema": "http://json-schema.org/draft-04/schema#",
    "title": "Schema to test names that would otherwise conflict with the path handling",
    "type":  "object",
    "properties": {
        "this/that": {
            "type": "object",
            "properties": {
                "t/h/e/ /o/t/h/e/r": {
                    "type": "object",
                    "properties": {
                        "man": {
                            "type": "string"
                        }
                    },
                    "required": ["man"]
                }
            },
            "required": ["t/h/e/ /o/t/h/e/r"]
        }
    }
}
```

Assume that you are validating the following JSON:

```
{
  "this/that": {
    "t/h/e/ /o/t/h/e/r": {}
  }
}
```

The error returned looks like:

```
{
  "code": "OBJECT_MISSING_REQUIRED_PROPERTY",
  "params": [
    "man"
  ],
  "message": "Missing required property: man",
  "path": "#/this~1that/t~1h/e/ /o/t/h/e/r"
}
```

If you use more than one slash in a key, you can no longer safely assume that the slash in the `path` variable is only used to join keys from different levels in the object hierarchy.

**Comment by zaggino** [comment]
@the-t-in-rtf should be fixed in `3.12.5`

**Comment by the-t-in-rtf** [comment]
Thanks again for the quick response, @zaggino.  Z-schema continues to be really useful for us, thanks for all the hard work.
